helm_deploy/values-dev.yaml

```diff
@@ -16,3 +16,4 @@ generic-service:
     WORKLOAD_SERVICE_URL: "https://hmpps-workload-dev.hmpps.service.justice.gov.uk"
     PROBATION_ESTATE_SERVICE_URL: "https://hmpps-probation-estate-api-dev.hmpps.service.justice.gov.uk"
     USER_PREFERENCE_SERVICE_URL: "https://hmpps-user-preferences-dev.hmpps.service.justice.gov.uk"
+    OASYS_URL: "https://ords.t1.oasys.service.justice.gov.uk"
```

Comment:
Our other services are in t3 (whatever that is). I don't suppose it matters a great deal as this is just a link to the homepage of the service.